samples/zigbee/light_switch/src/main.c

```diff
@@ -513,10 +583,14 @@ void main(void)
 	/* Start Zigbee default thread. */
 	zigbee_enable();
 
+#if CONFIG_BT_GATT_NUS
+	/* Initalize NUS command service. */
+	nus_cmd_init(on_nus_connect, on_nus_disconnect, commands);
+#endif /* CONFIG_BT_GATT_NUS */
+
 	LOG_INF("ZBOSS Light Switch example started");
 
 	while (1) {
-		dk_set_led(RUN_STATUS_LED, (++blink_status) % 2);
-		k_sleep(RUN_LED_BLINK_INTERVAL);
+		k_sleep(K_FOREVER);
```

Comment:
No LED blinks? :slightly_smiling_face:
This is fine from my perspective - please double check that the blinking was not documented in the example's documentation.